### PR TITLE
Update schedule

### PR DESCRIPTION
**Pull request by awesomehaze**
I accidentally pushed this to develop, oops

### CODE REVIEW

**Comment by awesomehaze** [comment]
Please review ✨✨✨

**Review by RoblKyogre — APPROVED** [review]
Look gud

File: src/schedule/index.ts [hunk 1]
```diff
@@ -66,11 +64,20 @@ export const schoolSpecialDates: any = {
...
+  // Presidents' Day weekend
+  '2 - 18 - 2022': Schedule.NONE,
+  '2 - 21 - 2022': Schedule.NONE,
 };
 
 export const msSpecialDates: any = {
   '12 - 9 - 2020': Schedule.REGULAR,
   '12 - 10 - 2020': Schedule.REGULAR,
+
+  '2 - 17 - 2022': Schedule.MINIMUM,
```
Comment (mark-crichton) [inline src/schedule/index.ts:80]: Doesn't seem to show on the schedule

**Review by mark-crichton — DISMISSED** [review]
Minimum schedule doesn't seem to show, everything else shows

**Review by MC-Atom — APPROVED** [review]
Minimum Schedule doesn't work for some reason but the assembly is coming up and I want to get this pulled soon. Everything else works. rip 7/8

**Review by MC-Atom — APPROVED** [review]
a

**Pull request merged**